### DISCUSSION TITLE
制作模板导出出现”在行1无法执行脚本。"PAG"未定”

### DISCUSSION THREAD

**Discussion by yishibanhuier** [Tencent/libpag#1624] [Q&A]
制作模板导出出现”在行1无法执行脚本。"PAG"未定”

**Comment by kevingpqi123** [comment]
这里具体的问题是啥，请将背景描述清楚以及复现步骤，只言片语无法定位问题

**Comment by yishibanhuier** [comment]
导出pag模板的时候出现这种报错  不知道什么原因 最后重新打开一下ae模板没有再出现了![微信截图_20230728141051](https://github.com/Tencent/libpag/assets/140788763/508a91b2-c124-46ff-aa13-aac1ea1e53d3)

**Reply by domchen** [reply]
如果可以复现的话，麻烦上传一个AE的文件，这样才可以定位问题并解决。

**Comment by kevingpqi123** [comment]
没有提供更进一步的信息，暂时关闭
